packages/server/src/controllers/agents.controller.ts

```
@@ -42,12 +57,90 @@ export class AgentsController {
   /**
    * Handle user request to a specific agent
    * @param userRequest - The user request containing agent ID and content
    * @returns Promise<AgentResponse> - Response with status and data
    */
+  @Post('upload-avatar')
+  async uploadAvatar(
+    @Headers('x-api-key') apiKey: string,
+    @Req() req: FastifyRequest
+  ) {
+    try {
+      const data = await (req as any).file();
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)